client/src/components/Header.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Header() {
+  return (
+    <h1>Header</h1>
```

Comment:
I would not write here `Header`. That's the biggest headline in your app. Google is looking and using that for `SEO`. I would pass here the name of the app or something like that.